examples/edge-stack-iso/custom/files/basic-setup.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# Pre-requisites. Cluster already running
+export RKE2KUBECTL="/var/lib/rancher/rke2/bin/kubectl"
+export K3SKUBECTL="/opt/bin/kubectl"
```

Comment:
Yes but we're providing specific examples here and not an automation that lots of customers will need. I believe being straight to the point with each example and only including the relevant bits and pieces for it is the best but I'd let others chime in on this.

Reply:
We only expect to support RKE2 for the CAPI/Metal3 case (management cluster and downstream clusters) so I would agree it may be simpler and less confusing to remove the k3s options for now?